src/App.tsx

```diff
@@ -22,36 +22,41 @@ const Container = styled.form`
 
 const App: React.FC = () => {
   const { sdk, safe } = useSafeAppsSDK();
-  const [selectedTab, setSelectedTab] = useState("dashboard");
+  const [selectedTab, setSelectedTab] = useState('dashboard');
   const module = useMemo(() => {
-    console.log("create module")
-    return new ethers.Contract(DelayedTxModule.address, DelayedTxModule.abi, new SafeAppsSdkProvider(safe, sdk));
-  }, [sdk, safe])
+    console.log('create module');
+    return new ethers.Contract(DelayedTxModule.address, DelayedTxModule.abi, new SafeAppsSdkSigner(safe, sdk));
```

Comment:
Couldn't we memoize the signer and use it in both the module and manager instance?